src/structs/Api.ts

```diff
@@ -43,8 +42,8 @@ export class Api extends EventEmitter {
 
     let url = `https://top.gg/api/${path}`
 
-    // @ts-ignore querystring typings are messed
-    if (body && method === 'GET') url += `?${qs.stringify(body)}`
+    // @ts-ignore URLSearchParams typings are records but the value here is a string.
+    if (body && method === 'GET') url += `?${new URLSearchParams(body)}`
```

Comment:
```suggestion
    if (body && method === 'GET') url += `?${new URLSearchParams(body).toString()}`
```

Reply:
It is already casted to a string as it is used in a template string.

Reply:
Wait nope hold on